paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -147,10 +153,12 @@ class PaparazziPlugin : Plugin<Project> {
 
       val recordTaskProvider = project.tasks.register("recordPaparazzi$variantSlug", PaparazziTask::class.java) {
         it.group = VERIFICATION_GROUP
+        it.description = "Record golden images for variant $variantSlug"
       }
       recordVariants.configure { it.dependsOn(recordTaskProvider) }
       val verifyTaskProvider = project.tasks.register("verifyPaparazzi$variantSlug", PaparazziTask::class.java) {
         it.group = VERIFICATION_GROUP
+        it.description = "Run screenshot tests for variant $variantSlug"
```

Comment:
```suggestion
        it.description = "Run screenshot tests for variant '${variant.name}'"
```

because variantSlug is capitalized